src/imgtools/ops/ops.py

```
@@ -1,31 +1,15 @@
-    BaseSubjectWriter,
     BaseWriter,
-    HDF5Writer,
-    ImageFileWriter,
-    MetadataWriter,
-    NumpyWriter,
-    SegNrrdWriter,
 )
-from imgtools.logging import logger
-from imgtools.modules.datagraph import DataGraph
-from imgtools.modules import Segmentation, map_over_labels, StructureSet
+from imgtools.modules import Segmentation, StructureSet, map_over_labels
```

Ruff (0.8.2)

`imgtools.modules.map_over_labels` imported but unused

Remove unused import: `imgtools.modules.map_over_labels`

(F401)